app/main.py

```diff
@@ -4,0 +1,34 @@
+import json
+from customer import Customer
+from shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app\\config.json") as f:
```

Comment:
Verify the path to the `config.json` file to ensure it is correct, as per the checklist requirement .